src/eatmemory.apple.c

```
@@ -1,32 +1,30 @@
+        return;
+    }
+    vm_statistics_data_t vm_stat;
 
-        count = sizeof(vm_stat) / sizeof(natural_t);
-        if (host_statistics(host_port, HOST_VM_INFO, (host_info_t)&vm_stat, &count) != KERN_SUCCESS) {
-            stats->supported = false;
-            return;
-        }
-        stats->supported = true;
+    count = sizeof(vm_stat) / sizeof(natural_t);
+    if (host_statistics(host_port, HOST_VM_INFO, (host_info_t)&vm_stat, &count) != KERN_SUCCESS) {
```

Cppcheck (2.20.0)

[error] Uninitialized variable

(uninitvar)